ci/tasks/deploy-frontend-with-adot.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Note this file is a copy of deploy-app.yml just with the addition of the adot variables